packages/desktop-client/src/components/settings/Format.js

```diff
@@ -16,52 +20,74 @@ let dateFormats = [
 ];
 
 export default function FormatSettings({ prefs, savePrefs }) {
-  function onDateFormat(e) {
-    let format = e.target.value;
+  function onDateFormat(format) {
     savePrefs({ dateFormat: format });
   }
 
-  function onNumberFormat(e) {
-    let format = e.target.value;
+  function onNumberFormat(format) {
     savePrefs({ numberFormat: format });
   }
 
+  function onHideFraction(e) {
+    let hideFraction = e.target.checked;
+    savePrefs({ hideFraction });
+  }
+
   let dateFormat = prefs.dateFormat || 'MM/dd/yyyy';
   let numberFormat = prefs.numberFormat || 'comma-dot';
 
   return (
-    <Section title="Formatting">
-      <Text>
-        <label htmlFor="settings-numberFormat">Number format: </label>
-        <select
-          defaultValue={numberFormat}
-          id="settings-numberFormat"
-          {...css({ marginLeft: 5, fontSize: 14 })}
-          onChange={onNumberFormat}
-        >
-          {numberFormats.map(f => (
-            <option key={f.value} value={f.value}>
-              {f.label}
-            </option>
-          ))}
-        </select>
-      </Text>
+    <>
```

Comment:
Do we really need a separate `Setting` block for each of these things? Could we just have a single one for all of these three local prefs?

Also: the dropdown seems to be "jumping" when its clicked:
![Kapture 2023-03-05 at 15 20 43](https://user-images.githubusercontent.com/886567/222969483-b9c98ba2-1f47-43d5-99f2-0f3f1dec0794.gif)